shop/src/pages/checkout/ChoosePayment.js

```diff
@@ -58,6 +60,7 @@ const reducer = (state, newState) => ({ ...state, ...newState })
 const ChoosePayment = () => {
   const history = useHistory()
   const { config } = useConfig()
+  const { shopConfig } = useShopConfig()
```

Comment:
This won't be available to non-logged in users. The front end should only be reliant on stuff in config.json, so we should probably add in a setting in there to indicate if offline payments are accepted or not

Reply:
Actually we can move the `offlinePaymentMethods` out of encrypted config to `config.json`. Don't think we have any reason not to do so. Since we are anyhow showing its content in the frontend

Reply:
Have moved this to the `config.json`. Nothing seems broken on local, gonna merge this to master